src/screens/ExchangeModal.tsx

```diff
@@ -251,16 +303,18 @@ export default function ExchangeModal({
     updateOutputAmount,
   } = useSwapInputHandlers();
 
-  const chainId = useMemo(
-    () =>
-      ethereumUtils.getChainIdFromType(
-        inputCurrency?.type || outputCurrency?.type
-      ),
-    [inputCurrency, outputCurrency]
-  );
+  const chainId = useMemo(() => {
+    if (inputCurrency?.type || outputCurrency?.type) {
+      return ethereumUtils.getChainIdFromType(
+        inputCurrency?.type! ?? outputCurrency?.type!
+      );
+    }
+
+    return 1;
+  }, [inputCurrency, outputCurrency]);
 
   const currentNetwork = useMemo(
-    () => ethereumUtils.getNetworkFromChainId(chainId || 1),
```

Comment:
@estebanmino @jxom Moved this default value to the function above as it seems like if they are both going to fallback then the Network and ChainId should be the same.

Reply:
makes sense, and maybe removing the `|| 1` part as well since the `getNetworkFromChainId` has a default value anyways